server/lib/data_import/my_anime_list/extractor/media.rb

```diff
@@ -0,0 +1,139 @@
+class DataImport::MyAnimeList
+  module Extractor
+    class Media
+      attr_reader :data
+
+      def initialize(json)
+        @data = JSON.parse(json)
+      end
+
+      def age_rating
+        rating = data['classification'].split(' - ')[0]
+
+        case rating
+        when 'G','TV-Y7' then :G
+        when 'PG', 'PG13' then :PG
+        when 'R', 'R+' then :R
+        when 'Rx' then :R18
+        end
+      end
+
+      def episode_count
+        data['episodes']
+      end
+
+      def episode_length
+        data['duration']
+      end
+
+      def synopsis
+        clean_desc(data['synopsis'])
+      end
+
+      def youtube_video_id
+        data['preview']
```

Comment:
On HB we currently store the _id_ of the youtube video, not the entire URL.  This should probably extract the youtube video ID from the URL.